sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/utils/string-utils.ts

```diff
@@ -0,0 +1,40 @@
+import { QueueProcessSummaryResult } from "../../../../../../../processors/models/processors.models";
+/**
+ * Converts an assessment id to its fixed text format.
+ * @param assessmentId assessment id to be converted.
+ * @returns converted assessment id.
+ */
+export function assessmentIdToText(assessmentId: number): string {
+  return assessmentId.toString().padStart(10, "0");
+}
+
+/**
+ * Converts a disbursement id id to its fixed text format.
+ * @param disbursementId disbursement id to be converted.
+ * @returns converted disbursement id.
+ */
+export function disbursementIdToText(disbursementId: number): string {
+  return disbursementId.toString().padStart(10, "0");
```

Comment:
Just wondering if the logic is the same for both, how about creating a reusable method something like, 
![image](https://github.com/bcgov/SIMS/assets/77353155/66b28b65-5b0e-4651-aa3c-bb2a6ed654b8)
?

Reply:
I will review the documentation. If there is a great amount of numbers expressed always as 10 characters long I will make the change 😉

Reply:
or just when calling the function, we can pass the `maxLength` .

Reply:
What I meant @ann-aot was, in the case that we need to always pass the value I would like to avoid using `numberToText(firstDisbursement.id, 10)` every time. The idea of having the specific methods was to have the `maxLength` centralized.
We have a good amount of numbers defined with a length of 10. If needed we can create a specific method using the `numberToText`. I've just changed the method.